crates/sncast/src/helpers/interactive.rs

```diff
@@ -0,0 +1,199 @@
+use crate::helpers::config::get_global_config_path;
+use anyhow::{anyhow, Context, Result};
+use camino::Utf8PathBuf;
+use configuration::search_config_upwards_relative_to;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::Select;
+use std::env::current_dir;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::Path;
+use toml_edit::{DocumentMut, Item, Table, Value};
+
+pub fn prompt_to_add_account_as_default(account: &str) -> Result<()> {
+    let mut options = Vec::new();
+
+    if let Some(option) = get_global_config_path().ok().map(|global_path| {
+        format!(
+            "Yes, global default account ({}).",
+            to_tilde_path(global_path.as_str())
+        )
+    }) {
+        options.push(option);
+    }
+
+    if let Some(option) = current_dir()
```

Comment:
nit: Also in other places
```suggestion
    if let Some(option) = env::current_dir()
```